style.css

```diff
@@ -12,6 +12,10 @@ body {
   color: #000;
   font-family: "Poppins", sans-serif;
   width: 400px;
+
+  /* padding-left: 120px;
+  padding-right: 120px; */
+
   margin: auto;
```

Comment:
Please do not leave behind commented pieces of code as this clutters the clarity of your code base.